notebooks/text_models/solutions/keras_for_text_classification.ipynb

```diff
@@ -29,23 +29,25 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.                Lambda(lambda x: tf.reduce_mean(x, axis=1)),  # TODO 4
</pre><p> </p><p><code>keras.ops.mean</code> in Keras.</p><p>Or we can replace the Lambda with <code>GlobalAveragePooling1D()</code> layer which does the same thing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/654/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>I will update to keras.ops.mean because it's a little more clear that it's a bag of words since GlobalAveragePool1D would use 1d conv windows. </p>